superset/contrib/docker/superset_config.py

```diff
@@ -152,3 +152,26 @@ def mutator(f):
 # This parameter documented as deprecated but frontend still uses it
 # Should match `--timeout` value of gunicorn
 SUPERSET_WEBSERVER_TIMEOUT = 300
+
+if get_env_variable('OAUTH_PROVIDER', False) == 'google':
```

Comment:
maybe we should exit with an error if `OAUTH_PROVIDER` value is unknown.

Reply:
I've done a draft implementation of that, although I don't love it